aliasing/api/combat.py

```diff
@@ -524,19 +532,26 @@ def id(self):
         """
         return self._group.id
 
-    def get_combatant(self, name):
+    def get_combatant(self, name, strict=None):
         """
-        Gets a :class:`~aliasing.api.combat.SimpleCombatant`, fuzzy searching (partial match) on name.
+       Gets a :class:`~aliasing.api.combat.SimpleCombatant` from the group.
 
         :param str name: The name of the combatant to get.
-        :return: The combatant.
+        :param strict: Whether combatant name must be a full case insensitive match.
+                       None  - strict and then partial match. Default.
+                       False - only partial match
+                       True  - only strict match
+        :return: The combatant or None.
         :rtype: :class:`~aliasing.api.combat.SimpleCombatant`
         """
         name = str(name)
-        combatant = next((c for c in self.combatants if name.lower() in c.name.lower()), None)
-        if combatant:
-            return combatant
-        return None
+        combatant = None
+
+        if strict or strict is None:
```

Comment:
`if strict is not False` is probably clearer (this is one of the only times where you actually use `is` to compare to a primitive, interestingly enough)